src/cli/tc.rs

```diff
@@ -60,51 +59,81 @@ pub fn run() -> Result<()> {
 }
 
 pub async fn init(file_path: &str) -> Result<()> {
-  let tailcallrc: resource::Resource<str> = resource_str!("examples/.tailcallrc.graphql");
-
-  let answer = Confirm::new("Do you want to add a file to the project?")
-    .with_default(false)
-    .prompt();
-
-  match answer {
-    Ok(true) => {
-      let file_name = inquire::Text::new("Enter the file name:")
-        .with_default(".graphql")
-        .prompt()
-        .unwrap_or_else(|_| String::from(".graphql"));
-
-      let file_name = format!("{}.graphql", file_name.strip_suffix(".graphql").unwrap_or(&file_name));
-
-      let confirm = Confirm::new(&format!("Do you want to create the file {}?", file_name))
-        .with_default(false)
-        .prompt();
-
-      match confirm {
-        Ok(true) => {
-          fs::write(format!("{}/{}", file_path, &file_name), "")?;
-
-          let graphqlrc = format!(
-            r#"|schema:
-               |- './{}'
-               |- './.tailcallrc.graphql'
-          "#,
-            &file_name
-          )
-          .strip_margin();
-          fs::write(format!("{}/.graphqlrc.yml", file_path), graphqlrc)?;
-        }
-        Ok(false) => (),
-        Err(e) => return Err(e.into()),
+  let folder_exists = fs::metadata(file_path).is_ok();
+
+  if !folder_exists {
+    let confirm = Confirm::new(&format!("Do you want to create the folder {}?", file_path))
+      .with_default(false)
+      .prompt();
+
+    match confirm {
+      Ok(true) => fs::create_dir(file_path)?,
+      Ok(false) => (),
+      Err(e) => return Err(e.into()),
+    };
+  }
+
+  let tailcallrc = include_str!("../../examples/.tailcallrc.graphql");
+
+  let file_name = ".tailcallrc.graphql";
+  let yml_file_name = ".graphqlrc.yml";
+  let yml_exists = fs::metadata(format!("{}/{}", file_path, yml_file_name)).is_ok();
+
+  if !yml_exists {
+    fs::write(format!("{}/{}", file_path, yml_file_name), "")?;
+
+    let graphqlrc = r#"|schema:
+         |- './.tailcallrc.graphql'
+    "#
+    .strip_margin();
+
+    fs::write(format!("{}/.graphqlrc.yml", file_path), graphqlrc)?;
+  }
+
+  let tailcall_exists = fs::metadata(format!("{}/{}", file_path, file_name)).is_ok();
+
+  if tailcall_exists {
+    // confirm overwrite
+    let confirm = Confirm::new(&format!("Do you want to overwrite the file {}?", file_name))
+      .with_default(false)
+      .prompt();
+
+    match confirm {
+      Ok(true) => fs::write(format!("{}/{}", file_path, file_name), tailcallrc.as_bytes())?,
+      Ok(false) => (),
+      Err(e) => return Err(e.into()),
+    };
+  } else {
+    fs::write(format!("{}/{}", file_path, file_name), tailcallrc.as_bytes())?;
+  }
+
+  let graphqlrc_path = format!("{}/.graphqlrc.yml", file_path);
```

Comment:
I think you already have this path created above. Let's check once again variable name to make code more readable